apps/i18n/common/en_us.json

```diff
@@ -469,8 +469,6 @@
   "congratsCertificateThanks": "Thanks for submitting!",
   "congratsCertificateContinue": "Now, keep going with our other courses, or see more options to Learn beyond an Hour below.",
   "congratsNextLevelHeading": "Graduate to the next level",
-  "congratsSelfPacedPlTitle": "Explore more self-paced professional learning modules",
-  "congratsSelfPacedPlDescription": "Keep learning with our robust selection of self-paced professional learning modules. Our self-paced learning works in tandem with our curriculum so you can empower yourself and your students at the pace that’s best for you.",
```

Comment:
It looks like there's still unintended diff here @mikeharv?

Reply:
Oh, hmm. Must be due to merging in Molly’s branch and staging. I’ll make sure it gets cleaned up.